requirements.txt

```diff
@@ -3,7 +3,7 @@ chardet==3.0.4 --hash=sha256:9f178988ca4c86e8a319b51aac1185b6fe5192328eb5a163c28
 furl==2.0.0 --hash=sha256:cc0eb8998dcc7c5b58bc8625891a9ff563e2765e112024fa3d1e3521481de8b6
 idna==2.7 --hash=sha256:954e65e127d0433a352981f43f291a438423d5b385ebf643c70fd740e0634111
 orderedmultidict==1.0 --hash=sha256:25489716d76d2cc8aa656bfb00cd40b6ca29d5e11ccde0db60c2b46ad52bb40a
-pyyaml==3.13 --hash=sha256:59fa6f097310e25248d5aa1d7b7e619ea22dfaada67e9c65262457b3e5b3a5c8
+pyyaml==5.1 --hash=sha256:b8d80623e9d4e348c59ea726ce3032a2eb15abca6a48d3828362d11c6014a0a7
```

Comment:
re: your note about verification of the sha256sum, isn't this the sha256sum of the built wheel, i.e. it's not available in https://github.com/freedomofpress/securedrop-debian-packaging/wiki/PyYAML-5.1?

Reply:
Nope, the verification was done on the source, the final builtwheel sha256sum (in signed format) is in here https://github.com/freedomofpress/securedrop-debian-packaging/pull/35

Reply:
Sorry for making this PR first.

Reply:
OK so yes this is the hash of the built wheel ;)